css/style.css

```diff
@@ -0,0 +1,369 @@
+@import url('/css/fonts.css');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: url(/img/Background.png);
+  background-size: cover;
+  font-family: 'Roboto Bold', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.container {
+  max-width: 70%;
+  margin: auto;
+}
+
+.game {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  align-items: center;
+}
+
+.menu {
+  width: 100%;
+  position: relative;
+}
+
+.level-choice__header {
+  margin-bottom: 16px;
+  font-size: 54px;
+  color: #FFFFFF;
+}
+
+.level-choice__instruction {
+  display: block;
+  margin-bottom: 60px;
+  font-size: 18px;
+  color: #FFFFFF;
+}
+
+.level-choice__wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-content: center;
+  align-items: start;
+  max-height: 300px;
+}
+
+.level {
+  position: relative;
+  display: block;
+  margin-bottom: 24px;
+  font-size: 48px;
+  line-height: 59px;
+  color: #FFFFFF;
+  margin-left: 46px;
+}
+
+.selected-level {
+  color: #D62B2E;
+}
+
+.level::before {
+  display: none;
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #D62B2E;
```

Comment:
Codacy found an issue: [Expected "#D62B2E" to be "#d62b2e" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)